EnhancedMediaPlayer/MediaPlayerViewController.swift

```diff
@@ -100,3 +127,10 @@ public class MediaPlayerViewController: UIHostingController<MediaPlayerView> {
     }
 }
 
+extension MediaPlayerViewController {
+    enum Constants {
+        static let barUpdateIntervalInMs: CGFloat = 1000
```

Comment:
Could we use a lower interval for "smoothness" ? How much of a performance hit it would be if it was lowered to something like 200ms ?

Reply:
If it's negligible we could have it at 16.6ms to keep up with the iPhone 60hz screen refresh rate